File: src/io/kyros/content/commands/all/afkstore.java
package io.kyros.content.commands.all;

import io.kyros.content.commands.Command;
import io.kyros.model.entity.player.Player;

public class afkstore extends Command {
    @Override
    public void execute(Player player, String commandName, String input) {
        player.setInTradingPost(false);
        player.getShops().openShop(195);
    }
}

File: data/afk_store.lua
require("player.command.command_manager")


local shopBuilder = _G.shopBuilder
local Right = luajava.bindClass("io.kyros.model.entity.player.Right")

local MessageBuilder = luajava.bindClass("io.kyros.model.entity.player.message.MessageBuilder")

local function afk_store(player, _)
    local mockEvent = {
        getNpcId = function() return AFK_STORE end,
        getOptionIndex = function() return 1 end,
        getPlayer = function() return player end
    }

    local message = luajava.new(MessageBuilder)
    message:text("This shop is underworks, please use ::newafkstore")
    message:send(player)
    --shopBuilder:onNpcOption(mockEvent)
end

CommandManager:registerCommand("afkstore", afk_store, Right.PLAYER)
